particle-accelerator/.erb/configs/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What is this `.erb` folder for? Is this the Electron stuff that you just copied?

Reply:
Yes, it's part of the boilerplate.